src/aiida_quantumespresso/parsers/parse_raw/pw.py

```diff
@@ -279,11 +279,12 @@ def detect_important_message(logs, line):
             logs.warning.append(message)
 
 
-def parse_stdout(stdout, input_parameters, parser_options=None, parsed_xml=None):
+def parse_stdout(stdout, input_parameters, parser_options=None, parsed_xml=None, crash=None):
```

Comment:
Think `crash_file` or `crash_content` would be a bit clearer.
```suggestion
def parse_stdout(stdout, input_parameters, parser_options=None, parsed_xml=None, crash_file=None):
```